content/docs/concepts/multicluster-configuration/index.md

```diff
@@ -0,0 +1,101 @@
+regardless of where the workloads are actually running. It's transparent
+to the application whether it's deployed in a single or multicluster mesh.
+To achieve this, all of the services need to be managed by a single logical
+control plane, although not necessarily a single physical Istio control plane.
+There are two possible deployment approaches:
+
+1. A single Istio control plane with access to and configuration for all the services in the mesh.
+
+1. Multiple synchronized Istio control planes with replicated service and routing configurations. 
+
+Even within these two topologies, there is more than one way to configure things.
+Which approach to use and how to configure it depends on the requirements of the application
+as well as the features and limitations of the underlying cloud deployment platform.
+
+### Single control plane topology
+
+This multicluster configuration uses a single Istio control plane,
+typically running on one of the clusters, whose Pilot
+manages services on both local and remote clusters and configures the Envoy sidecars
+for all of them. This approach works best in environments where all of the contributing
+clusters have VPN connectivity so every pod in the mesh is reachable from anywhere else using
+the same IP address.
+
+{{< image width="80%" ratio="36.01%"
+    link="./multicluster-with-vpn.svg"
+    caption="Istio mesh spanning multiple Kubernetes clusters with direct network access to remote pods over VPN"
+    >}}
+
+In this configuration, one or more clusters run a remote Istio configuration which connects them
+to a single Istio control plane, allowing it to manage all of the Envoy's as a single mesh.
+Detailed instructions for setting up this kind of multicluster topology can be found
+in [single control plane with vpn](/docs/setup/kubernetes/multicluster-install/vpn/).
+
+If setting up a VPN network to provide universal pod-to-pod connectivity is difficult or impossible,
+Istio Pilot can be configured to enable location-aware service routing (a.k.a. split-horizon EDS).
+When configured this way, a request from a sidecar in one cluster to a service that is
+also running in the same cluster will be forwarded to the local service IP as usual, but when the destination
+workload is running in a different cluster it will use the remote cluster Gateway IP to connect to the service instead.
+
+{{< image width="80%" ratio="36.01%"
+    link="./multicluster-split-horizon-eds.svg"
+    caption="Istio mesh spanning multiple Kubernetes clusters using single Istio control plane and Gateway to reach remote pods"
+    >}}
+
+The [single control plane with gateways](/docs/examples/multicluster/split-horizon-eds/) example provides
+instructions for experimenting with this feature.
+
+### Multiple control plane topology
+
+Instead of using a central Istio control plane to manage the mesh,
+in this configuration each cluster has an **identical** Istio control plane
+installation, each managing its own endpoints.
```

Comment:
At this point, the text has already established that the two topologies are different and how they differ. The conditional clause at the beginning of the sentence makes it about the contrast instead of about defining the multiple control plane topology's characteristics. Consider:
```suggestion
In a multiple control plane configuration, each cluster has an **identical** Istio control plane
installation and each control plane manages its own endpoints.
```